_posts/2016-05-04-folders-upcoming.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: Upcoming support for ROI Folders
+categories: data-model future-plans
+---
+
+For many months the OME team has been working on what will become a new
+2016 OME Data Model. The OME Data Model specifies Regions of Interest
+(ROIs) in terms of a set of Shapes. As OMERO 5.3 will use the new Data
+Model, the upcoming changes include an initial round of adjusting how
+the Data Model and OMERO each represent Shapes so that the two match
+better. They will each also include a significant addition: *Folders*, a
+new top-level model object.
+
+## Using ROI Folders
+
+Our initial focus is on supporting ROI-based Folder workflows. Both
+OMERO.insight and OMERO.web will offer some support for users to have
```

Comment:
Should we also mention CLI/scripts here as this becomes the main tool when organizing folders at scale.